python.d/README.md

```diff
@@ -2504,3 +2504,31 @@ apache_log:
 Module has preconfigured jobs for nginx, apache and gunicorn on various distros.
 
 ---
+
+# neo
+
+Displays key metrics for the Neo4j graph database (www.neo4j.com) 
+   * Total number of nodes
+   * number of nodes for the top5 labels
+   * number of nodes for the top5 relations
+
+Requires the neo4j-driver module (JS driver) to access the wanted server.
+Users can install the neo4j-driver package by running the /usr/libexec/netdata/python-modules-installer.sh script (with sudo) and restart the netdata service (sudo service netdata restart).
+
+By default, listens to a Neo4j instance on localhost, accessible with neo4j/password credentials. You can update the credentials in the  /etc/netdata/python.d/neo.conf file. 
+
+## configuration   
+See the /etc/netdata/python.d/neo.conf file to change the settings
+```
+neo:
+    name : 'neo'
+    host : 'localhost'
+    port : '7687' # port where Bolt protocol is listening
+    user : 'neo4j'
+    pwd : 'password'
+    update_every : 30
```

Comment:
i suggest to move `update_every` to module if you think 30 should be default

Reply:
I'll do that as maintenance when I'll understand this sentence and packages in python